integration-libs/cdc/user-profile/forgot-password/cdc-forgot-password-component.service.ts

```diff
@@ -59,8 +61,8 @@ export class CDCForgotPasswordComponentService
             },
             GlobalMessageType.MSG_TYPE_ERROR
           );
-          this.busy$.next(false);
         }
+        this.busy$.next(false);
```

Comment:
[MINOR]
Now we set `busy$` to false immediately when the function is called, without waiting for `this.cdcJsService.resetPasswordWithoutScreenSet` to finish.

[QUESTION]
Maybe(?) we could reuse the methods of the super class `ForgotPasswordComponentService`: `onSuccess()` and `onError()` https://github.com/SAP/spartacus/blob/69d81dd41713d2547a991f0a5c87a4030295610b/feature-libs/user/profile/components/forgot-password/forgot-password-component.service.ts#L65-L77? I'm not insisting on it, just asking. For example:
- in the subscription to `this.cdcJsService.resetPasswordWithoutScreenSet`, we could potentially call `onSuccess()` or `onError()`, depending on the result of the operation
- if `cdcLoaded` is `false`, we could also call `onError()`

Reply:
- regarding [MINOR] -  issue fixed in https://github.com/SAP/spartacus/pull/16160
- regarding [QUESTION] - we're not using super.onSuccess/onError, because we don't want to trigger the global message here (as it's already triggered from the CDC JS Service)